From 2e7b66dde144300febf26e95704b5581322dba77 Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Date: Mon, 4 Mar 2019 03:27:45 +0700
Subject: [PATCH 049/248] arm64: dts: renesas: r8a774c0-cat874: Add USB2.0 Host
 support

This patch adds USB2.0 Host support to the RZ/G2E board on the
R8A774C0 SoC.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts | 22 ++++++++++++++++++++++
 1 file changed, 22 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts b/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
index ae1c617..53d1758 100644
--- a/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
@@ -172,6 +172,11 @@
 			"audio_clkout_a", "audio_clkout1_a";
 		function = "audio_clk";
 	};
+
+	usb0_pins: usb {
+		groups = "usb0_b";
+		function = "usb0";
+	};
 };
 
 &scif2 {
@@ -315,3 +320,20 @@
 &ssi4 {
 	shared-pin;
 };
+
+&ehci0 {
+	dr_mode = "host";
+	status = "okay";
+};
+
+&ohci0 {
+	dr_mode = "host";
+	status = "okay";
+};
+
+&usb2_phy0 {
+	pinctrl-0 = <&usb0_pins>;
+	pinctrl-names = "default";
+
+	status = "okay";
+};
-- 
2.7.4

